### DISCUSSION TITLE
Lobe theme freezed about 2~4 seconds after fully load backend

### DISCUSSION THREAD

**Discussion by suonnon** [lobehub/sd-webui-lobe-theme#174] [General]
After fully loaded the backend and open on browser, my Lobe theme takes around 4 seconds to use. Before that, it looks like freeze. Is there a way to improve this delayed/laggy performance? 

Specs: Amd ryzen 7 2700x, RTX 3060 Ti, 32GB Ram DDR4, Ssd samsung 980 pro, Edge/Chrome/Brave browser

**Comment by canisminor1990** [comment]
try reduce animation mode in setting panel

**Reply by suonnon** [reply]
Yes, I tried to turn that on but no changes at all. It's only happen with Edge; Brave/Chrome are fine now
